### PR TITLE
General improvements

### PR DESCRIPTION
**Pull request by SteveMavic**
- improved HTTP Requests logging
- changed icons colors to match Kodi color pattern
- applied icons to "Tags", "Years", "Seasons" main menu list items
- improved readability which arts are being applied to main menu list item
- disabled set_content in some folders (because of this - was unable to set custom icons)
- added command descriptions #502 
- improved commands visibility, commands have own icon now

From:
obraz
obraz

To:
obraz
obraz

### CODE REVIEW

**Comment by bigretromike** [comment]
Not the biggest fun of thumbnails for "filters" but other than that I really like the cmd icon.
Looks like it's that time of year xD

File: api/common/__init__.py [hunk 1]
```diff
@@ -150,21 +147,24 @@ def call(self,
...
 
         elif call_type == APIType.PUT:
-            print(f"Api.PUT == {url} -- -H {headers} -D {data}")
             data = json.dumps(data).encode("utf-8")
             req = Request(url, data=data, headers=headers, method="PUT")
         else:
             print(f"Unknown === {url}")
+            debug(f"Unknown === {url}")
+
+        print(f"Sending HTTP request: {req.method} {req.get_full_url()}\r\nHeaders: {req.headers}\r\nData: {req.data}")
```
Comment (bigretromike) [inline api/common/__init__.py:164]: do we need duplicate information ? I understand that we can ditch print, or convert it to debug but both print same data :-)

**Review by bigretromike — APPROVED** [review]

File: api/common/__init__.py (hunk 1 above)
Comment (SteveMavic) [reply api/common/__init__.py:164]: We could get rid of prints, but prints do not appear in kodi.log unless set debug mode in Kodi, not plugin settings

**Comment by SteveMavic** [comment]
> Not the biggest fun of thumbnails for "filters" but other than that I really like the cmd icon. Looks like it's that time of year xD

Which one? Those icons were already in resources :D

**Comment by bigretromike** [comment]
@SteveMavic bump version ;-)

**Pull request merged**